reference/mongodb/mongodb/driver/writeconcern/getwtimeout.xml

```diff
@@ -10,7 +10,7 @@
  <refsect1 role="description">
   &reftitle.description;
   <methodsynopsis>
-   <modifier>final</modifier> <modifier>public</modifier> <type class="union"><type>int</type><type>MongoDB\BSON\Int64</type></type><methodname>MongoDB\Driver\WriteConcern::getWtimeout</methodname>
+   <modifier>final</modifier> <modifier>public</modifier> <type class="union"><type>int</type><type>null</type></type><methodname>MongoDB\Driver\WriteConcern::getWtimeout</methodname>
```

Comment:
This method never returns `null`. It only utilizes `RETURN_LONG` and the proto only notes an `int` return type. See: https://github.com/mongodb/mongo-php-driver/blob/1.14.0/src/MongoDB/WriteConcern.c#L231

Perhaps this was a mistake when you modified the stub in https://github.com/mongodb/mongo-php-driver/commit/6887226d34566cd772bbacb34e10fc7bd4000f57#diff-8c9ed4ed771a5941bf91127c022c00eec1817dd20b860a79c2c8f6335bee233eR34 to use `?int`?

Reply:
Interesting. Yes, that method definitely doesn't return `null`. I'll update the stub in a separate PR and have updated the docs.